packages/browser-test-runner/src/createWebpackConfig.ts

```diff
@@ -26,29 +34,17 @@ export const createWebpackConfig = (
                 ],
             },
             plugins: [
-                new NodePolyfillPlugin(),
-                new webpack.ProvidePlugin({
-                    process: 'process/browser'
+                new NodePolyfillPlugin({
```

Comment:
Question: Is the `process` polyfill which is now provided by `NodePolyfillPlugin` equal (or better) compared to the previous polyfill which was provided by the `webpack.ProvidePlugin`?